node/metrics/metrics.go

```diff
@@ -51,6 +52,8 @@ func InitBaseMetrics(statusHandlerUtils StatusHandlersUtils) error {
 	appStatusHandler.SetUInt64Value(common.MetricNumConnectedPeers, initUint)
 	appStatusHandler.SetUInt64Value(common.MetricEpochForEconomicsData, initUint)
 
+	appStatusHandler.SetInt64Value(common.MetricTrieSnapshotStartTime, initInt)
```

Comment:
I do not think this metric is useful. We know the snapshot starts at epoch change. I would rather log the snapshot duration.

Reply:
:+1: changed to last snapshot duration